server/cli/src/cli.rs

```diff
@@ -154,6 +154,10 @@ enum Action {
     },
     /// Will generate a plugin bundle
     GeneratePluginBundle(GeneratePluginBundle),
+    /// Will insert generated plugin bundle
+    InstallPluginBundle(InstallPluginBundle),
```

Comment:
Was thinking maybe we just need generate and install, kept this one however as a manual was to install via CLI